controllers/storagecluster/reconcile.go

```diff
@@ -532,6 +532,12 @@ func (r *StorageClusterReconciler) reconcilePhases(
 		}
 	}
 
+	// For security of encrypted messages, When both encryption and compression are enabled, compression setting will be ignored and message will not be compressed.
+	networkSpec := instance.Spec.Network
+	if networkSpec != nil && networkSpec.Connections != nil && networkSpec.Connections.Encryption != nil && networkSpec.Connections.Encryption.Enabled && networkSpec.Connections.Compression != nil && networkSpec.Connections.Compression.Enabled {
+		r.Log.Info("Both in-transit encryption & compression are enabled. To protect security of encrypted messages ceph will ignore the compression setting by default here. To override this behavior,refer https://docs.ceph.com/en/quincy/rados/configuration/msgr2/#confval-ms_compress_secure")
```

Comment:
Let's split this log line into multiple lines. Also, I don't think we should have a ref to the upstream doc here.

Reply:
@agarwal-mudit Can you clarify if it's appropriate to pass this ceph doc in the logs? for overriding the default? I believe What Nitin is saying is true & we shouldn't pass this as overriding what ceph does by default may cause problems.

Reply:
As discussed with Mudit, Removing the doc link.